test/validation/api/init/init_main.c

```diff
@@ -168,6 +168,40 @@ static void init_test_log_thread(void)
 	CU_ASSERT(ret == 0);
 }
 
+static void init_test_log_fn_get(void)
+{
+	int ret;
+	odp_instance_t instance;
+	odp_init_t param;
+
+	odp_init_param_init(&param);
+
+	ret = odp_init_global(&instance, &param, NULL);
```

Comment:
Could also test that `odp_log_fn_get()` returns non-`NULL` before initialization as per the spec.

Reply:
Added a check for this.